docs/concepts/copilot/copilot-ai-capabilities.md

```diff
@@ -0,0 +1,169 @@
+</div></div>
+<script src="https://fast.wistia.net/assets/external/E-v1.js" async></script>
+
+## Next-transformation suggestions
+
+Data Copilot can suggest the next transformation in a series or the next expression within a Gem.
+
+### Suggest Gems
+
+Data Copilot can suggest the next transformation in a series as long as there is an open Tail node.
+
+![Suggest Gems](img/copilot_next_suggestion.png)
+
+See the following Join suggestion example:
+
+1. Select a Dataset of interest.
+2. Data Copilot suggests Datasets which are frequently used with the selected Dataset.
+3. Data Copilot then suggests a next transformation, in this case, a Join Gem.
+
+### Suggest Expressions
+
+At the more granular level, Data Copilot can suggest expressions within Gems.
+
+![Suggest expressions](img/copilot_next_suggestion_expression.png)
+
+Within our [advanced Expression Builder](/Spark/expression-builder) you can:
+
+1. Type an English text prompt.
+2. Data Copilot generates a code expression for a particular column.
+3. Click to accept the code expression or try again with a different prompt.
+4. Run the Pipeline up to and including this Gem, and observe the resulting data sample.
+
+## Generate with AI
+
+Data Copilot can generate scripts, user-defined functions, or functions for macro within a Gem.
+
+<div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://fast.wistia.net/embed/iframe/vhf63tjnun?videoFoam=true" title="Modify an existing Pipeline" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+<script src="https://fast.wistia.net/assets/external/E-v1.js" async></script>
+
+## Map with AI
+
+You don't have to worry about mapping the schema across your Model. Data Copilot will map the target schema with the existing Gems and datasets.
+
+<div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://fast.wistia.net/embed/iframe/p5ab7sgfdw?videoFoam=true" title="Modify an existing Pipeline" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+<script src="https://fast.wistia.net/assets/external/E-v1.js" async></script>
+
+## Code with AI
+
+In addition to the visual editor above, you'll also see code suggestions in the code editor as below.
+
+![CodeSuggestions](./img/copilot_code_suggestion.png)
```

Comment:
We don't see "AskAI  | Generate  | FixIt" in the video below. 
Maybe remove the image above for consistency?

Reply:
Done.